source/agora/common/crypto/ECC.d

```diff
@@ -17,6 +17,8 @@
 
 module agora.common.crypto.ECC;
 
+import agora.common.crypto.Key;
+import agora.common.crypto.Schnorr;
```

Comment:
The import is because of `Pair` right ? If so I think it'd be better to move `Pair` here.

Reply:
Pair is only used in unittest.
I removed agora.common.crypto.Schnorr;

Reply:
I moved to Key.D